v3/setup/klam-ssh.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
```

Comment:
agreed. I'm also starting to lean towards `-xe` due to the amount of stuff going on in here